internal/common/foraging/fishhunt.go

```diff
@@ -0,0 +1,76 @@
+package foraging
+
+import (
+	"math"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/config"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+// FishHunt The teams that are involved and the resources they put in.
+type FishHunt struct {
+	Participants map[shared.ClientID]float64
+	Params       fishHuntParams
+}
+
+// fishHuntParams : Defines the parameters for the normal distibution for the fishing returns
+type fishHuntParams struct {
+	Mu    float64
+	Sigma float64
```

Comment:
The equivalent members in `deerHuntParams` aren't capitalised, since they aren't exported. Should we make this consistent? @lhl2617 @QFSW